### ISSUE TITLE
Run basecalling of multiple fast5 files in parallel?

### ISSUE THREAD

**Issue by Thomieh73** [NorwegianVeterinaryInstitute/Nepal#46]
Is it possible to run basecalling in parallel on a hpc cluster. 

Say each fast5 file is analyses in a seperate slurm job.
Question:
* does that create a sequencing summery file for each fast 5 file. Can I concatenate those into one file to extract?
* How does that work with demultiplexing, or should I do that with barcode_analyzer after merging all the basecalling results?
*

**Comment by Thomieh73** [comment]
No longer needed, since dorado will do that when given multiple GPUs
